code/modules/hydroponics/grown/corn.dm

```diff
@@ -89,5 +90,25 @@
 		user.put_in_hands(S)
 	snap_pops -= 1
 	if(!snap_pops)
-		new /obj/item/grown/corncob(user.loc)
+		new /obj/item/grown/snapcorncob(user.loc)
 		qdel(src)
+
+/obj/item/grown/snapcorncob
+	seed = /obj/item/seeds/corn/snapcorn
+	name = "snap corn cob"
+	desc = "A reminder of pranks gone by."
```

Comment:
Make this a subtype of `/obj/item/grown/corncob`?

Reply:
That's a fair point, I didn't even think of doing that tbh.